Digitall.Persistance.EF/Repositories/Repository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Digitall.Warehouse.Domain.Abstraction;
+using Microsoft.EntityFrameworkCore;
+
+namespace Digitall.Persistance.EF.Repositories
```

Comment:
As you said when I start working with Dapper I'll do it then. For now the file location is ok, because there is no need to use anywhere :)
I personally prefer to do things when it is need it than in advance.